templates/ruckus_fastiron_show_arp.template

```diff
@@ -0,0 +1,18 @@
+Value NUMBER (\S+)
+Value Required IP_ADDRESS ([A-Fa-f0-9:\.]+)
+Value Required MAC_ADDRESS (([A-Fa-f0-9\.]{14}|None))
+Value TYPE (\S+)
+Value AGE (\S+)
+Value Required PORT (\S+)
+Value STATUS (\S+)
+Value VLAN (\S+)
+
+Start
+  ^All\s+ARPs:\s+\d+,\s+maximum\s+capacity:\s+\d+
+  ^Total\s+(number\s+of\s+|)ARP\s+[Ee]ntries\s*:\s+\d+
+  ^Entries\s+in\s+default\s+routing\s+instance:
+  ^No\.\s+(IP\s+Address|IP)\s+(MAC\s+Address|MAC)\s+Type\s+Age\s+Port\s+Status(\s+VLAN|)
+  ^${NUMBER}\s+${IP_ADDRESS}\s+${MAC_ADDRESS}\s+${TYPE}\s+${AGE}\s+${PORT}\s+${STATUS}\s+${VLAN} -> Record

```

Comment:
Thanks for your submission!

What separate out each expected output into their own states since there is a small difference in the output (one will provide VLAN and the other doesn't.

```
Start
  ^No\.\s+IP\s+MAC\s+Type\s+Age\s+Port\s+Status\s+VLAN -> Type1
  ^No\.\s+IP\s+Address|\s+MAC\s+Addres\s+Type\s+Age\s+Port\s+Status -> Type2

Type1
  xxxx

Type2
  xxxx
```

Reply:
Changes made, 0 changes to output.